src/context/LocaleContext/LocalContext.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { createContext } from 'react'
```

Comment:
I assume the file should be called `LocaleContext.tsx` (Local**e** instead of Local), and that this is just a typo?